### ISSUE TITLE
Dark mode for this project

### ISSUE THREAD

**Issue by nitin-787** [nitin-787/mynotes#105]

**Comment by sarg19** [comment]
Hey, can you assign this issue to me?

**Comment by AnurupMondal** [comment]
Hello, I would like to work on this.

**Comment by nitin-787** [comment]
indeed

**Comment by HarshMakwana27** [comment]
Hey, I want to work on this

**Comment by nitin-787** [comment]
yes, why not here the UI link of figma currently it contains light mode only, but I added the dark mode example you can the mark the dark ui  based on that color palette: https://www.figma.com/file/gfaxLApOmJRt8lFrIRrWku/UNI-Share?type=design&node-id=1030%3A1456&t=xH1tCNuKpydSckpe-1

![Screenshot 2023-05-21 141403](https://github.com/nitin-787/mynotes/assets/76255199/1b8f995f-e095-45b1-96a1-acb989c06aa3)

**Comment by nitin-787** [comment]
are you still working on it? @HarshMakwana27

**Comment by HarshMakwana27** [comment]
I need some time.

**Comment by Supratim69** [comment]
Hii I'm willing to work on this. Can I get more info on this?
